src/less/section-notice/section-notice.less

```diff
@@ -68,14 +68,14 @@ span[role="region"].section-notice {
     grid-column: 1;
     grid-row: 1;
     height: @spacing-200;
+    margin-inline-end: @spacing-200;
     margin-top: 1px;
-    padding-right: @spacing-200;
 }
 
 .section-notice__main {
     grid-column: 1 / 3;
     grid-row: 1;
-    padding-right: @spacing-200;
+    margin-inline-end: @spacing-200;
```

Comment:
This changed from padding to margin. Just checking this is intentional.

Reply:
Yea, it was intentional. I realize it's not a 1:1 swap. I think I may have been the one to originally do padding here; it was probably not the right approach - spacing for static elements is probably better with `margin`. Using `padding` is probably better for interactive elements, especially when taking click targets into consideration.